<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>313</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>313</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>313</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HeadsUp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_today_title">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Tuesday 29th June</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit_today">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="lineWrapMode">
                 <enum>QPlainTextEdit::NoWrap</enum>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="plainText">
                 <string>15:00 - 15:30	Placeholder Text
15:00 - 15:30	J/C Monthly Something Meeting Pineapple
15:00 - 15:30	Placeholder Text</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_tomorrow_title">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Wednesday 30th June</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit_tomorrow">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="lineWrapMode">
                 <enum>QPlainTextEdit::NoWrap</enum>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="plainText">
                 <string>15:00 - 15:30	Placeholder Text
15:00 - 15:30	J/C Monthly Something Meeting Pineapple
15:00 - 15:30	Placeholder Text</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_calendar">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>120</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="tabKeyNavigation">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="rowCount">
             <number>5</number>
            </property>
            <property name="columnCount">
             <number>8</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>50</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>13</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>23</number>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>14</number>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <item row="0" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>Mon</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>Tue</string>
             </property>
            </item>
            <item row="0" column="3">
             <property name="text">
              <string>Wed</string>
             </property>
            </item>
            <item row="0" column="4">
             <property name="text">
              <string>Thu</string>
             </property>
            </item>
            <item row="0" column="5">
             <property name="text">
              <string>Fri</string>
             </property>
            </item>
            <item row="0" column="6">
             <property name="text">
              <string>Sat</string>
             </property>
            </item>
            <item row="0" column="7">
             <property name="text">
              <string>Sun</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>01/08</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>01/08</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>01/08</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>01/08</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_shortcut1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;This is a sample&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_shortcut2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_shortcut3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_shortcut4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_shortcut5">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_shortcut6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_shortcut8">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_shortcut8_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDeveloper">
    <property name="title">
     <string>Developer</string>
    </property>
    <widget class="QMenu" name="menuLog_Level">
     <property name="title">
      <string>Log Level</string>
     </property>
     <addaction name="actionDISABLE"/>
     <addaction name="actionWARNING"/>
     <addaction name="actionINFO"/>
     <addaction name="actionDEBUG"/>
     <addaction name="actionVERBOSE"/>
    </widget>
    <addaction name="menuLog_Level"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_manage_shortcuts"/>
    <addaction name="actionRemove_all_shortcuts"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUsing_the_calendar"/>
    <addaction name="actionCreating_shortcuts"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuDeveloper"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionDISABLE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DISABLE</string>
   </property>
  </action>
  <action name="actionWARNING">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WARNING</string>
   </property>
  </action>
  <action name="actionINFO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>INFO</string>
   </property>
  </action>
  <action name="actionDEBUG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DEBUG</string>
   </property>
  </action>
  <action name="actionVERBOSE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>VERBOSE</string>
   </property>
  </action>
  <action name="actionShow_developer_console">
   <property name="text">
    <string>Show developer console</string>
   </property>
  </action>
  <action name="actionCalendar_Notification_Manager">
   <property name="text">
    <string>Calendar Notification Manager</string>
   </property>
  </action>
  <action name="actionEdit_shortcuts">
   <property name="text">
    <string>Edit shortcuts...</string>
   </property>
  </action>
  <action name="actionClear_shotcuts">
   <property name="text">
    <string>Clear shotcuts...</string>
   </property>
  </action>
  <action name="action_manage_shortcuts">
   <property name="text">
    <string>Manage shortcuts</string>
   </property>
  </action>
  <action name="actionRemove_all_shortcuts">
   <property name="text">
    <string>Remove all shortcuts</string>
   </property>
  </action>
  <action name="action_manually_change_login">
   <property name="text">
    <string>Change login time...</string>
   </property>
  </action>
  <action name="actionChange_hours_per_day">
   <property name="text">
    <string>Change hours per day...</string>
   </property>
  </action>
  <action name="action_refresh_calendars">
   <property name="text">
    <string>Refresh calendars</string>
   </property>
  </action>
  <action name="actionUsing_the_calendar">
   <property name="text">
    <string>Calendar abbriviations...</string>
   </property>
  </action>
  <action name="actionCreating_shortcuts">
   <property name="text">
    <string>Creating shortcuts...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
